consent/strategy_default.go

```diff
@@ -409,10 +437,16 @@ func (s *DefaultStrategy) verifyAuthentication(w http.ResponseWriter, r *http.Re
 
 	if !session.LoginRequest.Skip {
 		if time.Time(session.AuthenticatedAt).IsZero() {
-			return nil, errorsx.WithStack(fosite.ErrServerError.WithHint("Expected the handled login request to contain a valid authenticated_at value but it was zero. This is a bug which should be reported to https://github.com/ory/hydra."))
+			return nil, errorsx.WithStack(fosite.ErrServerError.WithHint(
+				"Expected the handled login request to contain a valid authenticated_at value but it was zero. This is a bug which should be reported to https://github.com/ory/hydra."))
+		}
+
+		loginSession, err := flowctx.FromCookie[flow.LoginSession](ctx, r, s.r.FlowCipher(), flowctx.LoginSessionCookie)
+		if err != nil {
+			return nil, err
```

Comment:
Referencing the other comment in ConfirmLoginSession - here, we would get an error if the cookie is not found, correct? In that case, loginSession would always be defined, no?

Reply:
@hperl can you please add a comment to `ConfirmLoginSession`, clarifying the upsert statement?